src/devices/osram.ts

```diff
@@ -254,6 +254,13 @@ const definitions: Definition[] = [
         description: 'SMART+ Candle E14 Dimmable White',
         extend: [ledvanceLight({})],
     },
+    {
+        zigbeeModel: ['Control box TW'],
+        model: 'Control box TW',
```

Comment:
Could you use the model number that is written on the device here?

Reply:
Is there some EAN? E.g. on the site where you bought it.

Reply:
Finally was able to get the controller out and copy the model number. Please @Koenkk consider a re-review.